includes/gateways/stripe/class-give-stripe.php

```diff
@@ -59,6 +59,21 @@ public function __construct() {
 		 */
 		public function includes() {
 
+			// Include admin files.
```

Comment:
@mehul0810 are these admin files loading for frontend also? if yes then why?

Reply:
Loaded these files for admin only.